Makefile

```diff
@@ -881,6 +882,7 @@ ifneq ($(GIT_DIR),)
 endif
 
 CPP_PROTO_ROOT := $(LIBROACH_SRC_DIR)/protos
+CPP_PROTO_CCL_ROOT := $(LIBROACH_SRC_DIR)/protosccl
```

Comment:
Should this be `$(LIBROACH_SRC_DIR)/ccl/protos` instead? There's no reason to put it in the base directory.

Reply:
either way it kind of stutters. We can have `protosccl/ccl/...` or `ccl/protos/ccl/...`. I'm not sure I care enough about generated files though.